tools/doc/type-parser.js

```diff
@@ -87,11 +85,9 @@ module.exports = {
         }
 
         if (typeUrl) {
-          typeLinks.push('<a href="' + typeUrl + '" class="type">&lt;' +
-            typeTextFull + '&gt;</a>');
+          typeLinks.push(`<a href="${typeUrl}" class="type">&lt;${typeTextFull}&gt;</a>`);
```

Comment:
This makes the linter sad :-(

But we can make it happy by moving the arg to the next line like so:
```js
typeLinks.push(
  `<a href="${typeUrl}" class="type">&lt;${typeTextFull}&gt;</a>`);
```

Reply:
Yes you're right I change it right now

Reply:
@rmg - hope you are good with the changes now?

Reply:
assuming so, landing now.